sale_order_line_date/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# © 2016 OdooMRP team
+# © 2016 AvanzOSC
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# © 2016 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Sale Order Line Date",
+    "version": "12.0.1.0.0",
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
+              "Odoo Community Association (OCA)",
+    "website": "https://odoo-community.org/",
+    "category": "Sale",
+    "license": "AGPL-3",
+    # sale_order_dates became part of the sale module in 12.0
+    "depends": [
+        "sale_management",
```

Comment:
this module should depends in stock and it doesn't

In v11 depenede in `sale_order_dates` https://github.com/OCA/sale-workflow/blob/11.0/sale_order_line_date/__manifest__.py#L19

Reply:
V11 depends on sale_order_dates, but the commitment_date field on sale.order was moved to base sale module in V12. So no need for another dependency. 
The base sale module also has a method _prepare_procurement_values which is used to pass information to the picking. So there is no need for a dependency on stock.